src/main/java/jenkins/plugins/git/GitSCMSource.java

```diff
@@ -91,6 +109,42 @@ public boolean isIgnoreOnPushNotifications() {
     }
 
     @Override
+    public GitRepositoryBrowser getBrowser() {
+        return browser;
+    }
+
+    @Override
+    @DataBoundSetter
+    public void setBrowser(GitRepositoryBrowser browser) {
+        this.browser = browser;
+    }
+
+    @Override
+    public String getGitTool() {
+        return gitTool;
+    }
+
+    @Override
+    @DataBoundSetter
+    public void setGitTool(String gitTool) {
+        this.gitTool = gitTool;
+    }
+
+    @Override
+    public List<GitSCMExtension> getExtensions() {
+        if (extensions == null) {
+            extensions = new ArrayList<GitSCMExtension>();
+        }
+        return extensions;
```

Comment:
This method exposes internal implementation. FindBugs should not have accepted it. CC @MarkEWaite  (it it still disabled?)